pootle/static/js/common.js

```diff
@@ -203,6 +205,16 @@ PTL.common = {
       });
     });
 
+
+    /* Bind hotkeys */
+    const hotkeys = mousetrap(document.body);
+
+    hotkeys.bind('f1', (e) => {
```

Comment:
I think `F1` is a Windows convention. In web apps, I have always found <kbd>?</kbd> as the trigger. I am aware that <kbd>F1</kbd> also triggers while textareas/inputs have focus, although I find it a bit counter-intuitive as a non-Windows user. How about mapping <kbd>?</kbd> in addition to <kbd>F1</kbd>?